src/main/java/pl/tomaszdziurko/jvm_bloggers/TimeConstants.java

```diff
@@ -6,4 +6,7 @@
 
     public static final String EVERY_FRIDAY_AT_12_OCLOCK = "0 0 12 * * FRI";
 
+    public static final String EVERY_SIX_HOURS = "0 0 */4 * * *";
```

Comment:
*/4 clashes with 12 o'clock when newsletter is sent, It would be cool to have last refresh around 11 o'clock.